cohort_4/playbook/workshops/messenger/README.md

```diff
@@ -0,0 +1,1170 @@
+
+    var newLi = document.createElement("li");
+    newLi.innerHTML = usernameInputValue + " : " + messageInputValue;
+
+    var messageList = document.getElementById("messageList");
+    messageList.appendChild(messageElement);
+  }
+}
+```
+
+And then removing the `alert` messages
+
+```js
+window.onload = function() {
+  var submitButton = document.getElementById("submitButton");
+
+  submitButton.onclick = function() {
+    var messageInput = document.getElementById('messageInput');
+    var messageInputValue = messageInput.value;
+
+    var usernameInput = document.getElementById("usernameInput");
+    var usernameInputValue = usernameInput.value;
+
+    var newLi = document.createElement("li");
+    newLi.innerHTML = usernameInputValue + " : " + messageInputValue;
+
+    var messageList = document.getElementById("messageList");
+    messageList.appendChild(messageElement);
+  }
+}
+```
+
+## Huzzah! We're finished!
+
+<!--TODO: Celebrate Gif!-->
+
+## Extra Stuffs
+
+```
+Challenge Time!
+            __
+           / _)
+    .-^^^-/ /
+ __/       /
+ <__.|_|-|_|
+
+Instead of clicking a button in order to add a new message, how do you make it
+submit by pressing enter?
+```
+
+```js
+window.onload = function() {
+  var messageInput = document.getElementById("messageInput");
+
+  messageInput.onkeypress = function(e) {
+    if (e.keyCode === 13) { // this can be added later.
+      var messageInput = document.getElementById('messageInput');
+      var messageInputValue = messageInput.value;
+
+      var usernameInput = document.getElementById("usernameInput");
+      var usernameInputValue = usernameInput.value;
+
+      var newLi = document.createElement("li");
+      newLi.innerHTML = usernameInputValue + " : " + messageInputValue;
+
+      var messageList = document.getElementById("messageList");
+      messageList.appendChild(messageElement);
+    }
+  }
+}
+```
+
+```
+Challenge Time!
+            __
+           / _)
+    .-^^^-/ /
+ __/       /
+ <__.|_|-|_|
+
+Can you make it work if you pressed enter OR clicked on a button?
+```
+
+```js
+window.onload = function() {
+  var messageInput = document.getElementById("messageInput");
+  var submitButton = document.getElementById("submitButton");
+
+  messageInput.onkeypress = function(e) {
+    if (e.keyCode === 13) { // this can be added later.
+      submitMessage()
+    }
+  }
+
+  submitButton.onclick = function() {
+    submitMessage()
+  }
+
+  function submitMessage() {
+    var messageInput = document.getElementById('messageInput');
+    var messageInputValue = messageInput.value;
+
+    var usernameInput = document.getElementById("usernameInput");
+    var usernameInputValue = usernameInput.value;
+
+    var newLi = document.createElement("li");
+    newLi.innerHTML = usernameInputValue + " : " + messageInputValue;
+
+    var messageList = document.getElementById("messageList");
+    messageList.appendChild(messageElement);
+  }
+}
+```
```

Comment:
It doesn't... @zachlatta, how do I remove this? I'm not sure what to google, things I've thought of are:

- `removing a file from history`
- `removing accidental commit of file`

I'm not quite sure I understand the problem... I'm not sure how the messenger workshop got here...